public/app/features/provisioning/Shared/TokenPermissionsInfo.tsx

```diff
@@ -20,21 +20,22 @@ export function TokenPermissionsInfo() {
         <strong>"Fine-grained token".</strong>
         <Trans i18nKey="provisioning.token-permissions-info.make-sure">Make sure to include these permissions</Trans>:
```

Comment:
The colon at the end here outside the Trans tag was actually stopping the other text from being reported as untranslated, but it was disabled anyway so it didn't matter. But the rule has been updated to now catch this scenario